rest_framework_simplejwt/serializers.py

```diff
@@ -57,14 +67,14 @@ def validate(self, attrs):
         return {}
 
     @classmethod
-    def get_token(cls, user):
-        return cls.token_class.for_user(user)
+    def get_token(cls, user: AuthUser) -> Token:
+        return cls.token_class.for_user(user)  # type: ignore
```

Comment:
Same as with super(), this Serializer is used as a base for the other serializer, who set the token_class, but here it's still set to None. So the for_user() method can't be deduced.